lib/src/rendering/overflow_view.dart

```diff
@@ -239,18 +238,18 @@ class RenderOverflowView extends RenderBox
 
     if (showOverflowIndicator) {
       // We didn't layout all the children.
-      final RenderBox overflowIndicator = lastChild;
+      final RenderBox? overflowIndicator = lastChild;
       final BoxValueConstraints<int> overflowIndicatorConstraints =
           BoxValueConstraints<int>(
-        value: unrenderedChildCount,
-        constraints: childConstraints,
+        value: unRenderedChildCount,
+        constraints: childConstraints as BoxConstraints,
```

Comment:
We should instead make childConstraints a `BoxConstraints` instead of a `Constraints`.